numpy/testing/nosetester.py

```diff
@@ -326,9 +325,9 @@ def test(self, label='fast', verbose=1, extra_argv=None,
              <http://nedbatchelder.com/code/modules/coverage.html>`_).
         raise_warnings : str or sequence of warnings, optional
             This specifies which warnings to configure as 'raise' instead
-            of 'warn' during the test execution.  Valid strings are:
+            of 'always' during the test execution.  Valid strings are:
 
-              - "develop" : equals ``(DeprecationWarning, RuntimeWarning)``
+              - "develop" : equals ``(Warning,)``
```

Comment:
Let's leave it as is for the moment. If it causes too much trouble in beta we can always revert.